src/index.js

```diff
@@ -779,6 +779,21 @@ export default class ReactTable extends Methods(Lifecycle(Component)) {
         }}
         {...rootProps.rest}
       >
+        {showPagination.top ? (
```

Comment:
This JSX block should only be produced once, and used twice, instead of created twice